docs/application_migration.md

```diff
@@ -0,0 +1,16 @@
+## Checklist for migrating an Application to ArgoCD
+
+When migrating an application's deployment to be managed by ArgoCD use the following checklist to verify your process. 
+
+- [ ] Ensure your application manifests can be built using Kustomize.
+- [ ] If using secrets, make sure to include the .sops.yaml file in your repository.
+- [ ] Make sure to create the role granting access to namespace. See [here](cluster_ns_management.md#deploying-to-a-namespace) for more info.
+
+
+The following items require a PR to the `aicoe-cd` repository, and require an `aicoe-sre` team member to merge/deploy changes:
```

Comment:
Let's add this checklist as an App onboarding [PR template](https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository) (there can be multiple PR templates), that way it can guide the user more precisely.

Reply:
Multiple PR templates don't work the same as issue templates. In that, you do not have the templates accessible via the UI like you do for issues. You can access them by manipulating the PR url's, but this to me sounds not very helpful. Instead I just updated the current template to include the content, slightly reworded to fit the context better. Let me know what you guys think!